Change handling of "argument table" in `lousy/util.lua` from
deprecated (Lua 5.0) to modern (5.1 and up) syntax.

written by Stjujsckij Nickolaj

https://bugs.gentoo.org/show_bug.cgi?id=354355
https://github.com/mason-larobina/luakit/commit/99b1bb394beff0dd872dd822c930d412b98d00a9
--- a/lib/lousy/util.lua
+++ b/lib/lousy/util.lua
@@ -98,9 +98,9 @@ end
 -- @return A new table containing all keys from the arguments.
 function table.join(...)
     local ret = {}
-    for i = 1, arg.n do
-        if arg[i] then
-            for k, v in pairs(arg[i]) do
+    for i, tbl in pairs({...}) do
+        if tbl then
+            for k, v in pairs(tbl) do
                 if type(k) == "number" then
                     rtable.insert(ret, v)
                 else
-- 
1.7.4.rc3

